rust/src/treefile.rs

```diff
@@ -340,16 +343,18 @@ impl Treefile {
         let parsed = treefile_parse_recurse(filename, arch, 0)?;
         Treefile::validate_config(&parsed.config)?;
         let dfd = openat::Dir::open(filename.parent().unwrap())?;
-        let rojig_spec = if let &Some(ref rojig) = &parsed.config.rojig {
-            Some(Treefile::write_rojig_spec(&workdir, rojig)?)
+        let (rojig_name, rojig_spec) = if let Some(rojig) = parsed.config.rojig.as_ref() {
+            (Some(Treefile::write_rojig_spec(&workdir, rojig)?),
+             Some(CUtf8Buf::from_string(rojig.name.clone())))
```

Comment:
Shouldn't the ordering in this tuple be inverted?

Reply:
Yep, hadn't tested it locally, did so now, also ran rustfmt :arrow_down: